skt/__init__.py

```diff
@@ -1,99 +0,0 @@
-# Copyright (c) 2017 Red Hat, Inc. All rights reserved. This copyrighted
-# material is made available to anyone wishing to use, modify, copy, or
-# redistribute it subject to the terms and conditions of the GNU General
-# Public License v.2 or later.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT
-# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
-# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
-# details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software Foundation, Inc.,
-# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
-"""Functions used by multiple parts of skt."""
-from email.errors import HeaderParseError
-import email.header
```

Comment:
The email.header should be moved with the `get_patch_name` function, not `get_patch_mbox` one. Can you move it to that commit please?